jobs/ci-kubernetes-e2e-gce-large-manual-down.env

```diff
@@ -0,0 +1 @@
+# place-holder
```

Comment:
thanks to @zmerlynn, I think we've cleaned up gke, but haven't gone through gce yet. It's fine for now, I can handle it later.